### ISSUE TITLE
Как оформлять пояснения к формулам?

### ISSUE THREAD

**Issue by dvarubla** [latex-g7-32/latex-g7-32#84]
В принципе, здесь есть команда `\eqrem`, но новому стандарту она не соответствует, поскольку «где» в пояснениях не с абзаца (6.8.2). Вообще, я не очень понял, как это сделать по ГОСТ. Нужно ли делать пояснения так, как на рисунке?

![formula_test](https://user-images.githubusercontent.com/18435217/40641846-501e6832-6345-11e8-96ee-bed3df10772f.png)

Если да, тогда можно убрать весь код связанный с `\eqrem`, если же «c», и «b» должны быть под «a», то нужно как-то переделывать.

**Comment by LSinev** [comment]
```
\[
a = cb
\]
где \(a\) "--- неизвестная;\\
\(c\) "--- ещё неизвестная;
```

**Comment by dvarubla** [comment]
@LSinev у вас не по стандарту, нужен отступ перед «где» (и номер формулы). Да, в моём варианте не хватает математического режима для символов, но это и так ясно.

**Comment by einhander** [comment]
Как я понял `eqrem` сделан для соответствия ГОСТ 2.105 на который 7.32 изначально  ссылался, там действительно в примере обозначения идут один под другим.
![123](https://user-images.githubusercontent.com/5439042/40650923-15ea22a8-633d-11e8-8785-c0f866467445.png)
Сейчас этой ссылки  нет и примера оформления  подписи к формулам тоже нет.
Мое мнение что на данный момент `eqrem` можно выкинуть. 
Но с подписями есть одна проблема, при использовании equation и других float окружений подпись может быть отделена от самой формулы, что некрасиво, да и не по ГОСТ.
Для себя я решил путем
> \usepackage{newfloat}
> \DeclareFloatingEnvironment[placement={h},name=Equation]{eqndesc}

в преамбуле.
Мне кажется имеет смысл сделать подобный функционал, но не в самом стиле, а в примере, разместив этот или подобный код в преамбуле.

**Comment by dvarubla** [comment]
@einhander согласен, что можно тогда выкинуть `eqrem`, но разве `equation` — это float окружение? По-моему, уравнения всегда находятся в конкретном указанном месте. Чтобы подпись не отделялась от уравнения, можно использовать [что-то подобное](https://tex.stackexchange.com/questions/94699/absolutely-definitely-preventing-page-break), наверно.

**Comment by einhander** [comment]
@dvarubla вы правы, equation не float  окружение, что-то даже не обращал на это внимание.  Я сделал небольшой пример иллюстрирующий оба подхода.
Уравнение  с подписью как float:
![float](https://user-images.githubusercontent.com/5439042/40771894-40c53054-64c7-11e8-965a-2cb1b1ef9b03.png)
Уравнение  с подписью как не float объект:
![nofloat](https://user-images.githubusercontent.com/5439042/40771916-50b504e4-64c7-11e8-9799-1b241831131e.png)
Собственно подписи и уравнения могут быть большими и если они не float то на листе остается большое пустое пространство, если они не помещаются. Поэтому мне нравится float подход к подписям. 
Но какого хрена `newfloat` делает абзацный отступ равным 0.
Пока решил это через `\captionsetup` (не на скрине),  подвинув весь текст подписи на 12.5mm.

**Comment by dvarubla** [comment]
@einhander по-моему, легко найти в поисковике, что нужно самому задать `\parindent` и `\parskip`. Это можно сделать, например, так:
```latex
\DeclareFloatingEnvironment[placement={h},name=Equation]{eqndescNoIndent}
\edef\fixEqndesc{\noexpand\setlength{\noexpand\parindent}{\the\parindent}\noexpand\setlength{\noexpand\parskip}{\the\parskip}}
\newenvironment{eqndesc}{%
\begin{eqndescNoIndent}%
\fixEqndesc%
}
{\end{eqndescNoIndent}}
```
После этого, в `eqndesc` появляются абзацные отступы и интервалы между абзацами.

**Comment by einhander** [comment]
@dvarubla Спасибо!!! Видимо не там гуглил)))
Тогда сделаю PR с таким решением.

**Comment by konstantin-morenko** [comment]
На ctan.org есть пакет [eqexpl](https://ctan.org/pkg/eqexpl).  Думаю, он подходит для такого выравнивания.

**Comment by einhander** [comment]
@konstantin-morenko большое спасибо, то что нужно.
